src/executor/drop_executor.cpp

```diff
@@ -43,22 +43,48 @@ bool DropExecutor::DExecute() {
   const planner::DropPlan &node = GetPlanNode<planner::DropPlan>();
   DropType dropType = node.GetDropType();
   switch (dropType) {
+    case DropType::DB: {
```

Comment:
I know that you didn't write all of this code, but can you break this up the logic for the different drop targets into separate functions? See #1003 and my attempt to do this in `CreateExecutor` here:

https://github.com/cmu-db/peloton/pull/1005/commits/958681019845e35598b7cc7c4d06e63c881aaf1d

Reply:
Sure, I would change these.